MatrixOps.hpp

```diff
@@ -34,7 +34,36 @@ void initMatConst(Mat mat, unsigned char value) {
     }
 }
 
-void multMat(Mat A, Mat B, Mat C) {
+//implement C = A + B, dim of A should be equal to dim of B
+
+void addMat(Mat A, Mat B, Mat &C) {
```

Comment:
This definition should be `addMat(const Mat& A, const Mat& B, Mat& C)` for memory efficiency.